src/index.js

```diff
@@ -545,29 +545,18 @@ export default class extends Component {
   }
 
   renderScrollView = pages => {
-    if (Platform.OS === 'ios') {
```

Comment:
Awesome work dude). I wanted to create same pull request, and I really need this to be merged)
But you also need to remove code for android at line 396, or you will get an error.